docker/zarr-to-loom/optimus_zarr_to_loom.py

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/env python3
+import os
+import sys
+
+import zarr
+import loompy
+import scipy as sc
+from scipy.sparse import coo_matrix
+import argparse
+
+
+# Custom Exception
+class UnexpectedInputFormat(Exception):
+    pass
+
+
+def main():
+    # Parse the arguments
+    description = """This script converts ZARR optimus output to loom format"""
+    parser = argparse.ArgumentParser(description=description)
+    parser.add_argument('--input-zarr', dest="input_zarr_path", required=True, help="Path to input ZARR file")
+    parser.add_argument('--output-loom', dest="output_loom_path", required=True, help="Path to output loom file")
+    parser.add_argument('--sample_id', dest="sample_id", required=True, help="Sample identifier")
```

Comment:
command-line arguments should consistently use `-` or `_` as separators

Reply:
done